src/ScriptCs.Core/ScriptInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public class ScriptInfo : IScriptInfo
+    {
+        public ScriptInfo()
+        {
+            LoadedScripts = new List<string>();
+        }
+
+        public string ScriptPath { get; set; }
+        public IList<string> LoadedScripts { get; private set; }
```

Comment:
I think maybe this can even be `HashSet<string>` since there is no need to show the same script twice